app/services/search/feed_content.rb

```diff
@@ -18,6 +20,14 @@ def search_documents(params:)
         paginate_hits(hits, params)
       end
 
+      INCLUDED_CLASS_NAMES.each do |class_name|
+        define_method("#{class_name.underscore.pluralize}_document_count") do
+          Search::Client.count(
+            index: self::INDEX_ALIAS, body: count_filter(class_name),
+          ).dig("count")
+        end
+      end
```

Comment:
I was just venting about metaprogramming, but this is actually awesome 🎉 Great thinking! And this is a great way to have this "live alongside" `document_count`.